src/main/java/uk/gov/companieshouse/certificates/orders/api/validator/CompanyStatus.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.companieshouse.certificates.orders.api.validator;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum CompanyStatus {
```

Comment:
Investigate how dissolved companies are handled. Test in postman.

Reply:
With company status equals "dissolved" certificates document is created OK. 

![Screenshot 2021-11-18 at 19 19 19](https://user-images.githubusercontent.com/84074363/142482995-bd587aed-088d-4a7e-a27d-97a8e1b1b0a5.png)